### PR TITLE
Various frontend changes

### PR DESCRIPTION
**Pull request by jstavh**
Various frontend changes

### CODE REVIEW

**Comment by robinbb** [comment]
@jstavh Please assign to me when ready for me to take action.

**Comment by jstavh** [comment]
> @jstavh Please assign to me when ready for me to take action.

@robinbb It is ready, thank you.

File: web/app/page.tsx [hunk 1]
```diff
@@ -11,12 +11,12 @@ const Page = async () => {
...
     <div className="container relative">
       <PageHeader className="hidden md:block pb-6">
-        <PageHeaderHeading>Mina Improvement Proposals</PageHeaderHeading>
+        <PageHeaderHeading>Mina On-Chain Voting</PageHeaderHeading>
         <PageHeaderDescription>View and track the progress of Mina Improvement Proposals (MIPs).</PageHeaderDescription>
       </PageHeader>
 
       <PageHeader className="block md:hidden pb-6 text-center">
-        <PageHeaderHeading className="text-2xl">Mina Improvement Proposals</PageHeaderHeading>
+        <PageHeaderHeading className="text-2xl">Mina On-Chain Voting</PageHeaderHeading>
```
Comment (robinbb) [inline web/app/page.tsx:19]: No objection to this, but why is the same text and code repeated here? Violates DRY.

File: web/components/layout-footer.tsx [hunk 2]
```diff
@@ -14,16 +14,6 @@ export const Footer = () => {
           >
             Granola
           </a>
-          . The source code is available on{' '}
```
Comment (robinbb) [inline web/components/layout-footer.tsx:17]: The deletion of the period at the beginning of this line was a mistake.

**Review by robinbb — CHANGES_REQUESTED** [review]

File: web/app/page.tsx (hunk 1 above)
Comment (jstavh) [reply web/app/page.tsx:19]: @robinbb repeated where? Thought we wanted mina.vote top left, Mina On-Chain Voting here, then below keep the sentence.

File: web/components/layout-footer.tsx (hunk 2 above)
Comment (jstavh) [reply web/components/layout-footer.tsx:17]: @robinbb addressed in new commit below.

File: web/app/page.tsx (hunk 1 above)
Comment (robinbb) [reply web/app/page.tsx:19]: 5 lines above, there's another identical PageHeaderHeading.

Comment (robinbb) [reply web/app/page.tsx:19]: ... It seems to have to do with how the differently sized sites are coded.

**Review by robinbb — APPROVED** [review]

**Pull request merged**